core/services/job/runner_integration_test.go

```diff
@@ -345,8 +342,8 @@ ds1 -> ds1_parse;
 		os.MaxTaskDuration = models.Interval(cltest.MustParseDuration(t, "1s"))
 		err = jobORM.CreateJob(context.Background(), &os, os.Pipeline)
 		require.NoError(t, err)
-		var jb job.SpecDB
-		err = db.Preload("OffchainreportingOracleSpec", "id = ?", os.ID).
```

Comment:
actually not sure how this was working in gorm v1 - os.ID is a specDB.ID not an OffchainreprotingOracleSpec.ID. In v2 this fails as expected when running multiple subtest in parallel against the same db.